### ISSUE TITLE
IntelliSense, Peek Definition/Declaration and Reference lookup not working

### ISSUE THREAD

**Issue by Aeonos** [microsoft/vscode-cpptools#1003]
Hi there,
i dont get any information on variables, structures or anything while coding C on a Debian installation.
Debian 9.1 stretch x86
GLIBC 2.24
gcc 6.3
There are no problems reported in VS Code. If try to peek Definition (-> nothing happens). peek Declaration (nothing happens), goto Declaration (nothing happens), goto Definition (nothing happens), Find All References ( reports: no results). This is true for structs and variables i declared and also for function or anything from the standard libs.

I thought that the settings were not correct so i checked the c_cpp_properties.json file, especially the include paths.

```
{
    "configurations": [
        {
            "name": "Mac",
            "includePath": [
                "/usr/include",
                "/usr/local/include",
                "${workspaceRoot}"
            ],
            "defines": [],
            "intelliSenseMode": "clang-x64",
            "browse": {
                "path": [
                    "/usr/include",
                    "/usr/local/include",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            }
        },
        {
            "name": "Linux",
            "includePath": [
                "/usr/include",
                "/usr/local/include",
                "${workspaceRoot}/src",
                "${workspaceRoot}"
            ],
            "defines": [],
            "intelliSenseMode": "clang-x64",
            "browse": {
                "path": [
                    "/usr/include",
                    "/usr/local/include",
                    "${workspaceRoot}/src",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            }
        },
        {
            "name": "Win32",
            "includePath": [
                "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/include",
                "${workspaceRoot}"
            ],
            "defines": [
                "_DEBUG",
                "UNICODE"
            ],
            "intelliSenseMode": "msvc-x64",
            "browse": {
                "path": [
                    "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/include/*",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            }
        }
    ],
    "version": 2
}
```

That also seems fine to me. Only thing that keeps me thinking is the intelliSenseMode, which can only be set to clang-x64. Is it a problem , i am using a 32 bit OS?

I am writing a growing project and it would be really nice to have all the nice feature vs code usually provides.

**Comment by bobbrow** [comment]
We do not currently distribute 32-bit binaries for linux. #424 tracks this.

**Comment by sean-mcmanus** [comment]
Fixed with 0.14.4.
